docs/clients/p4rt-ctl.rst

```diff
@@ -319,7 +319,61 @@ Details about the supported commands
     ``COUNTER_FLOW``: Refers to the counter ID (generated ID by p4c, refer to bfrt.json file)
     and the counter table index. Format "counter_id=<number>,index=<number>"
 
-16. Get flow dump entries ::
+
+16. Add a rule for direct counter into the forwarding pipeline::
+
+    $ p4rt-ctl add-entry SWITCH TABLE FLOW
```

Comment:
There are two other `p4rt-ctl add-entry` commands in this file. Why do we need 3 different versions of `add-entry` command documented?

Reply:
we need this "add-entry" for direct counter  and in CLI we are showing the same under action

Reply:
Removed